mkdocs.yml

```diff
@@ -34,6 +34,7 @@ nav:
         - Reusable Data Validation Process: our_work/sde_data_validation.md
         - RAG: our_work/ds251_RAG.md
         - Primary Care Data Generator: our_work/swpclab.md
+        - Cancer high-risk cohorts: our_work/cancer_high_risk_cohorts.md
```

Comment:
it also needs adding to the navigation under the "Main work Areas" below, would you mind doing that?